std/meta.d

```diff
@@ -77,6 +78,17 @@
 
 module std.meta;
 
+/**
+ * Create a wrapper struct for a sequence of zero or more aliases. This
+ * can be used to pass multiple sequences as template arguments, because
+ * this prevent sequences from being coalesced together.
```

Comment:
Mind the D style - https://dlang.org/dstyle.html#phobos_documentation

...
- Every public function should have a Ddoc description and documented Params: and Returns: sections (if applicable):
- Documentation comments should not use more than two stars /** or two pluses /++ in the header line.
- Either Block comments (/**) or nesting block comments (/++) should be used except when the ddoc comment is a ditto comment such as /// Ditto
- Documentation comments should not have leading stars on each line.
...

Reply:
I know. At some point we agreed that's nice to have documentation written in the same format, but we didn't touch all the existing code (mainly to have clearer git blames), but we try to use the "consistent" format for at least new symbols.